### PR TITLE
gcc majors and clang deprecated in travis

### PR DESCRIPTION
**Pull request by lasote**
#2167 
I've reviewed the docs, nothing concrete to update, in #433 I updated a detail about package_tools that can be already merged to master.

### CODE REVIEW

File: conans/client/cmd/new_ci.py [hunk 1]
```diff
@@ -159,12 +159,13 @@
 """
 
 
-def get_build_py(name, user, channel, shared):
```
Comment (lasote) [inline conans/client/cmd/new_ci.py]: Unused params removed

File: conans/client/cmd/new_ci.py [hunk 2]
```diff
@@ -176,9 +177,8 @@ def get_travis(name, version, user, channel, linux_gcc_versions, linux_clang_ver
             config.append(linux_config_clang.format(version=clang, name=clang.replace(".", "")))
 
     xcode_map = {"8.1": "8.3",
-                 "8.0": "8.2",
                  "7.3": "7.3",
-                 "9.0": "9"}
+                 "9.0": "9.2"}
```
Comment (lasote) [inline conans/client/cmd/new_ci.py]: apple-clang 8.0 is no longer supported, xcode 8.2 comes with apple-clang 8.1

Comment (lasote) [reply conans/client/cmd/new_ci.py]: And apple-clang 9.0, also in xcode 9.2, so, better to keep it updated.

File: conans/client/cmd/new_ci.py [hunk 3]
```diff
@@ -247,7 +248,7 @@ def ci_get_files(name, version, user, channel, visual_versions, linux_gcc_versio
     if gitlab_clang_versions is True:
         gitlab_clang_versions = clang_versions
     if osx_clang_versions is True:
-        osx_clang_versions = ["7.3", "8.0", "8.1", "9.0"]
+        osx_clang_versions = ["7.3", "8.1", "9.0"]
```
Comment (memsharded) [inline conans/client/cmd/new_ci.py]: Just to make sure: apple-clang was not modified in latest release, we don't have info about ABI compatibility for apple-clang, the version scheme (4.0.1, 5.0.X) is just for clang

**Pull request merged**